bedrock/firefox/views.py

```diff
@@ -649,7 +652,7 @@ def get_template_names(self):
 class WhatsNewChinaView(WhatsnewView):
     def get_template_names(self):
         template = super().get_template_names()
-        if template == ['firefox/whatsnew/whatsnew-fx82.html']:
+        if template == ['firefox/whatsnew/whatsnew-fx83.html']:
```

Comment:
This shouldn't be changed. The special view for China was specific to WNP82 since that page promoted VPN, which we can't promote in China, we don't need to update it for 83 (and when WNP82 is eventually deleted we won't need this view at all).